content/tutorials/manage-the-rippled-server/configuration/configure-history-sharding.md

```diff
@@ -33,7 +33,7 @@ path=/var/lib/rippled/db/shards/nudb
 max_size_gb=50
 ```
 
-**Tip:** Ripple recommends using NuDB for the shard store (`type=NuDB`). NuDB uses fewer file handles per shard than RocksDB. RocksDB uses memory that scales with the size of data it stores, which may require excessive memory overhead. However, NuDB is designed to be used with SSD drives and does not work with rotational disks.
+The `type` field **must** be NuDB. [New in: rippled 1.3.0][]
```

Comment:
The `type` field is no longer required and can be omitted. If specified, it must be NuDB.